rrfs-test/validated_yamls/templates/obtype_config/atms_n21.yaml

```diff
@@ -0,0 +1,590 @@
+               sensor: atms_n21
+               channels: *atms_n21_channels
+               clwbkg_function:
+                 name: CLWRetFromBkg@DerivedMetaData
+               clwobs_function:
+                 name: CLWRetFromObs@DerivedMetaData
+               scatobs_function:
+                 name: SIRetFromObs@DerivedMetaData
+               clwmatchidx_function:
+                 name: CLWMatchIndex@DerivedMetaData
+                 channels: *atms_n21_channels
+               obserr_function:
+                 name: InitialObsError@DerivedMetaData
+                 channels: *atms_n21_channels
+               obserr_clearsky:  [ 4.500,  4.500,  4.500,  2.500,  0.550,
+                                   0.300,  0.300,  0.400,  0.400,  0.400,
+                                   0.450,  0.450,  0.550,  0.800,  4.000,
+                                   4.000,  4.000,  3.500,  3.000,  3.000,
+                                   3.000,  3.000]
+
+       - filter: Perform Action
+         filter variables:
+         - name: brightnessTemperature
+           channels: *atms_n21_channels
+         action:
+           name: inflate error
+           inflation variable:
+             name: ObsErrorFactorSituDepend@DerivedMetaData
+             channels: *atms_n21_channels
+
+       # Step 10: Gross check
+       # Remove data if abs(Obs-HofX) > absolute threhold
+       - filter: Variable Assignment
+         assignments:
+         - name: ObsErrorFactorLat@DerivedMetaData
+           type: float
+           function:
+             name: ObsErrorFactorLatRad@ObsFunction
+             options:
+               latitude_parameters: [25.0, 0.25, 0.04, 3.0]
+
+       - filter: Variable Assignment
+         assignments:
+         - name: ObsErrorBound@DerivedMetaData
+           channels: *atms_n21_channels
+           type: float
+           function:
+             name: ObsErrorBoundMW@ObsFunction
+             channels: *atms_n21_channels
+             options:
+               sensor: atms_n21
+               channels: *atms_n21_channels
+               obserr_bound_latitude:
+                 name: ObsErrorFactorLat@DerivedMetaData
+               obserr_bound_transmittop:
+                 name: ObsErrorFactorTransmitTop@DerivedMetaData
+                 channels: *atms_n21_channels
+                 options:
+                   channels: *atms_n21_channels
+               obserr_bound_topo:
+                 name: ObsErrorFactorTopo@DerivedMetaData
+                 channels: *atms_n21_channels
+               obserr_function:
+                 name: InitialObsError@DerivedMetaData
+                 channels: *atms_n21_channels
+                 threhold: 3
+               obserr_bound_max: [4.5, 4.5, 3.0, 3.0, 1.0,
+                                  1.0, 1.0, 1.0, 1.0, 1.0,
+                                  1.0, 1.0, 1.0, 2.0, 4.5,
+                                  4.5, 2.0, 2.0, 2.0, 2.0,
+                                  2.0, 2.0]
```

Comment:
Lines 534-538: change to:
```
               obserr_bound_max: *atms_n21_obserr_bound_max
```

Reply:
Ah you mean `atms_n20.yaml` does not use `&atms_n20_obserr_bound_max`?  If so, we will need to fix it. Thanks!

Reply:
"obserr_bound_max" can not be removed from the yaml file, which used in obsfunctions/ObsErrorBoundMW.cc  to do the background check.